.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -4,35 +4,32 @@ about: Create a report to help us improve
 title: ''
 labels: ''
 assignees: ''
-
 ---
 
-**Describe the bug**
-A clear and concise description of what the bug is.
+**Describe the bug** A clear and concise description of what the bug is.
```

Comment:
Prettier in action here. You can see all the files that prettier formatted here, [`a6ff147` (#21)](https://github.com/netlify/remix-compute/pull/21/commits/a6ff147c02c1c96483e4fb79afb486f4073dfd2a)